v3.0/how-to/maintain/identify-slow-queries.md

```diff
@@ -6,170 +6,190 @@ aliases: ['/docs-cn/sql/slow-query/']
-
-TiDB 会将执行时间超过 [slow-threshold](/reference/configuration/tidb-server/configuration-file.md#slow-threshold) 的语句默认单独输出到 [slow-query-file](/reference/configuration/tidb-server/configuration-file.md#slow-query-file) 文件中 ，并对慢日志的格式做了兼容，可以用 `pt-query-digest` 直接分析慢日志文件。`slow-threshold` 可以通过配置文件修改，默认是 300ms。`slow-query-file` 默认是 `tidb-slow.log`。
-
-## 示例
+## 日志示例
 
 ```sql
-# Time: 2019-04-25-15:06:54.247985 +0800
-# Txn_start_ts: 407942204372287489
+# Time: 2019-08-14T09:26:59.487776265+08:00
+# Txn_start_ts: 410450924122144769
 # User: root@127.0.0.1
-# Conn_ID: 2
-# Query_time: 2.6649544670000003
-# Process_time: 0.066 Wait_time: 0.014 Request_count: 8 Total_keys: 20008 Process_keys: 20000
+# Conn_ID: 3086
+# Query_time: 1.527627037
+# Process_time: 0.07 Request_count: 1 Total_keys: 131073 Process_keys: 131072 Prewrite_time: 0.335415029 Commit_time: 0.032175429 Get_commit_ts_time: 0.000177098 Local_latch_wait_time: 0.106869448 Write_keys: 131072 Write_size: 3538944 Prewrite_region: 1
 # DB: test
 # Is_internal: false
-# Digest: edb16a8f28d9c48790925fd1c868fdae3feb49bc58481dda7df228625a5ba6e1
-# Stats: t_slim:407941901863354370,t_wide:407941920305971202
-# Cop_proc_avg: 0.00825 Cop_proc_p90: 0.013 Cop_proc_max: 0.013 Cop_proc_addr: 127.0.0.1:22160
-# Cop_wait_avg: 0.00175 Cop_wait_p90: 0.002 Cop_wait_max: 0.002 Cop_wait_addr: 127.0.0.1:22160
-# Mem_max: 195802
-select count(1) from t_slim, t_wide where t_slim.c0>t_wide.c0 and t_slim.c1>t_wide.c1 and t_wide.c0 > 5000;
+# Digest: 50a2e32d2abbd6c1764b1b7f2058d428ef2712b029282b776beb9506a365c0f1
+# Stats: t:pseudo
+# Num_cop_tasks: 1
+# Cop_proc_avg: 0.07 Cop_proc_p90: 0.07 Cop_proc_max: 0.07 Cop_proc_addr: 172.16.5.87:20171
+# Cop_wait_avg: 0 Cop_wait_p90: 0 Cop_wait_max: 0 Cop_wait_addr: 172.16.5.87:20171
+# Mem_max: 525211
+# Succ: false
+insert into t select * from t;
 ```
 
-## 字段解析
+## 字段含义说明
+
+> 注：慢查询日志中所有时间相关字段的单位都是 **“秒”**
+
+Slow Query 基础信息：
 
 * `Time`：表示日志打印时间。
-* `Txn_start_ts`：表示事务的开始时间戳，也是事务的 ID，可以用这个值在日志中 grep 出事务相关的日志。
-* `User`：表示执行语句的用户名。
-* `Conn_ID`：表示 connection ID，即 session ID, 可以用类似 `con:3` 的关键字在 TiDB 日志中 grep 出 session ID 为 3 的日志。
-* `DB`：表示当前的 database。
-* `Index_ids`：表示语句涉及到的索引的 ID。
-* `Is_internal`：表示是否为 TiDB 内部的 SQL 语句。`true` 表示 TiDB 内部执行的 SQL 语句，比如 `analyze`，`load variable` 等；`false` 表示用户执行的 SQL 语句。
+* `Query_time`：表示执行这个语句花费的时间。
+* `Query`：表示 SQL 语句。慢日志里面不会打印 `Query`，但映射到内存表后，对应的字段叫 `Query`。
 * `Digest`：表示 SQL 语句的指纹。
-* `Memory_max`：表示执行期间 TiDB 使用的最大内存空间，单位为 byte。
-* `Query_time`：表示执行这个语句花费的时间。只有执行时间超过 slow-threshold 的语句才会输出这个日志，单位是秒，以下所有的时间字段的单位都是秒。
-* `Process_time`：执行 SQL 在 TiKV 的处理时间之和，因为数据会并行的发到 TiKV 执行，这个值可能会超过 `Query_time`。
-* `Wait_time`：表示这个语句在 TiKV 的等待时间之和，因为 TiKV 的 Coprocessor 线程数是有限的，当所有的 Coprocessor 线程都在工作的时候，请求会排队；当队列中有某些请求耗时很长的时候，后面的请求的等待时间都会增加。
+* `Txn_start_ts`：表示事务的开始时间戳，也是事务的唯一 ID，可以用这个值在 TiDB 日志中查找事务相关的其他日志。
+* `Is_internal`：表示是否为 TiDB 内部的 SQL 语句。`true` 表示 TiDB 系统内部执行的 SQL 语句，`false` 表示用户执行的 SQL 语句。
+* `Index_ids`：表示语句涉及到的索引的 ID。
+* `Succ`：表示语句是否执行成功。
```

Comment:
`Succ` is really weird, and not necessarily associated with Success. We don't normally segment a word like this.   Can we use the full form (success or succeed)?

Reply:
Agreed. But it appears in the log file now, so I prefer to not change the field name. We can optimize it later in the `tidb` repo and update this document in the future.